includes/post-hooks.php

```
@@ -27,391 +27,169 @@
-				return;
+		if ( $story_id && in_array( $status, array( StoryStatus::Sent->value, StoryStatus::Error->value ), true ) ) {
+			if ( ! $send_to_babbel ) {
+				$message = __( 'Story deleted from Babbel', 'zw-knabbel-wp' );
+				$context = 'checkbox_disabled';
+			} elseif ( 'trash' === $post->post_status ) {
+				$message = __( 'Story deleted (post trashed)', 'zw-knabbel-wp' );
+				$context = 'post_trashed';
+			} else {
+				$message = __( 'Story deleted (post unscheduled)', 'zw-knabbel-wp' );
+				$context = 'post_unscheduled';
 			}
```

PHPMD (2.15.0)

[error] The method sync_story_state uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)